pkg/app/app_test.go

```diff
@@ -686,7 +686,8 @@ releases:
 		{label: "name!=grafana", expectedReleases: []string{"grafana", "zipkin", "mongodb"}, expectErr: false},
 	}
 
-	defer env.Patch(t, ExperimentalEnvVar, ExperimentalSelectorExplicit)()
+	os.Setenv(ExperimentalEnvVar, ExperimentalSelectorExplicit)
+	defer os.Unsetenv(ExperimentalEnvVar)
```

Comment:
Nice! I agree we shouldn't add another dependency just for a small utility function 👍